src/ui/ui-kit/FormFieldset/FormFieldset.tsx

```diff
@@ -64,9 +66,12 @@ export const FormFieldset = React.forwardRef<
         }}
       >
         <VStack gap={4} style={{ width: '100%' }}>
-          <UIText kind="small/regular" as="label" ref={labelRef}>
-            {title}
-          </UIText>
+          <HStack gap={8} justifyContent="space-between" alignItems="center">
+            <UIText kind="small/regular" as="label" ref={labelRef}>
```

Comment:
Not sure if we need label here as well. However, will not change it here for now.